### ISSUE TITLE
[feature request] Number of Clients per SSID

### ISSUE THREAD

**Issue by zxyz** [mdlayher/unifi_exporter#16]
Hello,

I run a setup where I have different user groups seperated by SSIDs on the same site. I'd really like to have a feature to show the number of clients on each SSID, e. g. :

unifi_stations{ssid="A"} 110
unifi_stations{ssid="B"} 23

For my usecase a further differentation in sites would no be necessary. But the more sustainable implementation would probably be:

unifi_stations{site="X",ssid="A"} 110
unifi_stations{site="X",ssid="B"} 23
unifi_stations{site="Y",ssid="A"} 12
unifi_stations{site="Y",ssid="C"} 16

Thanks for considering!

**Comment by mdlayher** [comment]
Makes sense to me. I'll look into it and see if it's exposed by the controller!

**Comment by zxyz** [comment]
it is shown int the controller under "statistics", I've no idea if it's exposed via the api.
would still be really great if you could implement it :)

**Comment by akramer** [comment]
This data does appear to be visible via the API, in "vap_table" as a per-frequency per-SSID metric called "num_sta".
